recipe/meta.yaml

```diff
@@ -27,28 +28,31 @@ requirements:
     - charset-normalizer >=2.0.0,<2.1.0
     - idna >=2.5,<4
     - urllib3 >=1.21.1,<1.27
-    # no longer required for python 3.6+, but optional: https://github.com/psf/requests/pull/5797
+    # replaced by charset normalizer and is optional as of python 3.6+: https://github.com/psf/requests/pull/5797
     # - chardet >=3.0.2,<5
 
 test:
   requires:
     - pip
-    - python <3.10
+    - python >=3.6
```

Comment:
We will have to modify the python pinnings and only add the python restrictions in the skip section

Reply:
Since Python 3.6 is outside our support envelope, I do not believe adding a skip section is necessary. Unless I'm mistaken, this pinning is unnecessary and should be removed.

Reply:
I would defer to your team's standards, but I'm pretty sure it is necessary to remove the python pin altogether, and let the cbc.yaml determine the python version matrix. If a package is incompatible with one of the python versions in the matrix, that's when a skip value is used.